gitness.advisories.yaml

```diff
@@ -0,0 +1,29 @@
+schema-version: "2"
+
+package:
+  name: gitness
+
+advisories:
+  - id: CVE-2018-20699
+    events:
+      - timestamp: 2023-10-01T13:12:29Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
+          note: This vulnerability applies to the Docker daemon program, not the docker client libraries.
+
+  - id: CVE-2020-13401
+    events:
+      - timestamp: 2023-10-01T13:11:53Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
+          note: This vulnerability applies to the Docker daemon program, not the docker client libraries.
+
+  - id: GHSA-cf6v-9j57-v6r6
```

Comment:
We could use the CVE ID here: `CVE-2023-3515`

Reply:
Good catch!